app/res/layout/setup_install.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We can't use folders, so I'd like to make sure we're namespacing these components with prefixes like

fragment_setup_install